src/components/Bookform.js

```diff
@@ -38,19 +38,23 @@ const BookForm = ({ onAdd }) => {
 
   return (
     <form onSubmit={handleSubmit}>
+      <h4>ADD NEW BOOK</h4>
       <input
+        className="titleInput"
         type="text"
         placeholder="Title"
         value={title}
         onChange={(e) => setTitle(e.target.value)}
       />
       <input
+        className="authorInput"
         type="text"
         placeholder="Author"
         value={author}
         onChange={(e) => setAuthor(e.target.value)}
       />
       <select
+        className="dropdown"
         value={category}
         onChange={(e) => setCategory(e.target.value)}
       >
```

Comment:
- [ ] Nice styling of the general app. The rest of the app meets the styling replication, but the form part can use some improvement. We are to replicate the design so the position and size of the elements should fit. The first image is your form. The second one is the design's form.

  - Kindly move the "ADD NEW BOOK" text to the same position as the design.
  - Kindly increase the length of the inputs to be just like the design. As you can see the entire length of the form stretches to the length of the grey horizontal line above it.

  ![image](https://github.com/George7h/bookstores/assets/81468675/9f05d077-1539-48e8-a18a-46f25ad96d9c)

  ![image](https://github.com/George7h/bookstores/assets/81468675/4512d0e8-1065-4e2c-9c3a-6dd80cf07061)

Reply:
Hi @julie-ify,

Thank you for the review. I really appreciate it. I have one question... What about the Author? By not adding an input for the Author the entire book details won't be shown in the list. What do you suggest?